src/components/stepper-item/stepper-item.tsx

```diff
@@ -193,7 +203,17 @@ export class StepperItem implements InteractiveComponent {
     updateHostInteraction(this, true);
   }
 
+  disconnectedCallback(): void {
+    disconnectLocalized(this);
+  }
+
   render(): VNode {
+    numberStringFormatter.setOptions({
```

Comment:
Is there another hook we can use to update the options? You could watch for effectiveLocale, but that wouldn't handle changes to the parent's numbering system. Would that be a common scenario, you think?

Reply:
Yeah I'm not really sure there is a use case for changing locale and numberingSystem besides testing/debugging. I've just been updating the options to be safe lol. 

I guess we could create an internal event or a mutation observer for the numberingSystem prop on the parent and a watcher for effectiveLocale, if you think that's better

Reply:
I'm leaning towards only updating the formatter on `effectiveLocale` changes for now.